api/v1alpha1/function_types.go

```diff
@@ -51,6 +51,7 @@ type FunctionSpec struct {
 	FuncConfig   *Config                     `json:"funcConfig,omitempty"`
 	Resources    corev1.ResourceRequirements `json:"resources,omitempty"`
 	SecretsMap   map[string]SecretRef        `json:"secretsMap,omitempty"`
+	Env          map[string]string           `json:"env,omitempty"`
```

Comment:
Is it okay if move this `Env` part to `PodPolicy` struct? and it is suggested to use `[]corev1.EnvVar` so it is aligned with Kubernetes APIs.

Reply:
Thanks for the suggestion. I was actually thinking of doing that but was unsure how y'all would feel about it.

I'll switch to that.